doc/RULES

```diff
@@ -28,11 +28,19 @@ natively.  Those additional commands may also be useful on their own.
 -8	reject this rule unless current hash type uses 8-bit characters
 -s	reject this rule unless some password hashes were split at loading
 -p	reject this rule unless word pair commands are currently allowed
+-R	reject this rule unless stacked rules will also be applied later
+-S	reject this rule if stacked rules will also be applied later
 -u	reject this rule if an internal codepage is configured (doc/ENCODINGS)
 -U	reject this rule unless an internal codepage is configured
 ->N	reject this rule unless length N or longer is supported
 -<N	reject this rule unless length N or shorter is supported (--min-length)
 
+Note that -R will only accept a rule when both --rules (or --single) and
+--rules-stack are used at the same time (rules stacked on rules) and even then
+it will only accept lines from --rules, not from --stacked-rules.  If you run
```

Comment:
What's `--stacked-rules`? If you used this "colloquially", I think you shouldn't - when we list something as if it were a command-line option, it should actually be one.

Reply:
That was just a typo, will fix.

BTW I was never happy with the name "--rules-stack" but couldn't come up with anything better and there are pros with calling it "--rules-something". I'd rather have a syntax to use with `--rules=` for stacking "any" number of rulesets.